packages/mobile/src/screens/AxiosExample.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, ScrollView } from 'react-native';
+import News from '../components/News';
+import axios from 'axios';
+
+class AxiosExample extends Component{
+  constructor(props){
+		super(props);
+		this.state = {
+			news: []
+		}
+	}
+	componentDidMount() {
+    axios.get('https://newsapi.org/v2/everything?q=netflix&sortBy=publishedAt&apiKey=c04fd04b98d841d9ac3f0826f2497094')
+			.then(res => {
+				this.setState({ news: res.data.articles });
+			})
+  }
+  render(){
+    return(
+      <View style={styles.container}>
+        <ScrollView >
+          {
+            this.state.news.map((items, Id) =>
+              <News key={Id} data={items} />
+            )
+          }
+        </ScrollView>
+      </View>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
po co Ci styl skoro jest pusty

Reply:
Jeżeli tablica news będzie jeszcze do czegoś używana to warto zrobić destrukturyzacje